src/extension.ts

```diff
@@ -468,9 +470,15 @@ async function explain(obj, field) {
             swaggerSpecPromise = explainer.readSwagger();
         }
 
-        swaggerSpecPromise.then((s) => {
-            resolve(explainer.readExplanation(s, ref));
-        });
+        swaggerSpecPromise.then(
+            (s) => {
+                resolve(explainer.readExplanation(s, ref));
+                vscode.window.showErrorMessage('in after swagger spec promise');
+            },
+            (err) => {
+                vscode.window.showErrorMessage(`Explain failed: ${err}`);
```

Comment:
This seems like it could fire every time we try to explain an object, which if the promise has failed could result in a lot of noise.  Can we confirm that this fires only once?

Reply:
Better yet, I nulled out the promise so that we'll try again to load it on the error case...